linalg/hypre_parcsr.hpp

```diff
@@ -9,8 +9,8 @@
 // terms of the GNU Lesser General Public License (as published by the Free
 // Software Foundation) version 2.1 dated February 1999.
 
-#ifndef MFEM_HYPRE_PARCSR
-#define MFEM_HYPRE_PARCSR
+#ifndef MFEM_HYPRE_PARCSR_FILE
+#define MFEM_HYPRE_PARCSR_FILE
```

Comment:
I think the `_FILE` suffix should be replaced with `_HPP`.

Reply:
ok